plugin/audit_log/tests/mtr/audit_log_memory_consumption.test

```diff
@@ -0,0 +1,25 @@
+--source include/not_embedded.inc
+--source include/have_perfschema.inc
+
+let $MYSQLD_DATADIR= `select @@datadir`;
+let $log_file=$MYSQLD_DATADIR/test_audit.log;
+
+--remove_file $log_file
+SET GLOBAL audit_log_flush=ON;
+
+--disable_query_log
+--disable_result_log
+UPDATE performance_schema.setup_instruments
+SET ENABLED = 'YES'
+WHERE NAME LIKE 'memory/%';
+CREATE TABLE test (col1 longblob NOT NULL);
+# 10M query
+let $very_long_text=`SELECT REPEAT('a', 10485760)`;
+eval INSERT INTO test (col1) VALUES ('$very_long_text');
+--enable_query_log
+--enable_result_log
+--let $assert_text = Memory usage is less than 100M
+--let $query_result = query_get_value(SELECT current_alloc FROM sys.memory_global_by_current_bytes WHERE event_name = 'memory/sql/THD::variables', current_alloc, 1)
+--let $assert_cond= "$query_result" <= 100
+--source include/assert.inc
+DROP TABLE test;
```

Comment:
@altmannmarcelo  Should there be some kind of disabling performance schema instrumentation  after you did
```
UPDATE performance_schema.setup_instruments
SET ENABLED = 'YES'
WHERE NAME LIKE 'memory/%';
```
`mtr-check` may not detect this but it's possible that other tests running on the same mysqld instance will be affected.

Reply:
@percona-ysorokin I've checked other tests that change setup_instruments and the clean-up is enable everything:
few examples:
https://github.com/percona/percona-server/blob/8.0/mysql-test/suite/sysschema/include/ps_setup_instruments_cleanup.inc
https://github.com/percona/percona-server/blob/8.0/mysql-test/suite/perfschema/include/memory_aggregate_cleanup.inc#L40